.pipelines/e2e.yml

```diff
@@ -58,6 +58,20 @@ jobs:
         displayName: Enable regression tests in CI
         condition: in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI')
 
+      - script: |
```

Comment:
Is it easy for us to rename this pipeline/azp run target to something like /azp run e2e-wimi or something similar rather than `/azp run e2e`? We can handle it in a future PR as well, if that's easier.

This was raised as something that could cause confusion for contributors who want to re-run CI e2e as normal but end up kicking off a MIWI e2e instead.

Reply:
i'm looking into that. currently i'm unsure how to rename a pipeline.

Reply:
Renaming the pipeline isn't possible from this PR, but instead has to happen in the azure pipelines ui. I don't have permission to do so, but i'll find someone who does and get this renamed!

Reply:
I created a new ticket to address the renaming:
https://issues.redhat.com/browse/ARO-15569

Renaming the Pipeline can happen independently from this PR though, so we should be good to move forward